saltgui/static/scripts/api.js

```diff
@@ -116,24 +119,24 @@ class API {
   }
 
   isAuthenticated() {
-    return window.sessionStorage.getItem("token") !== null;
+    // As there is not an API call to check if you are authenticated we call
+    // the stats call to see if we can access that
+    return this._callMethod("GET", "/stats", {})
```

Comment:
normally, a function like `_fetch` would just react on no sessionkey or when 301 returned.
it should be possible to eliminate `isAuthenticated` completely.